app/code/Magento/PageCache/view/frontend/templates/inline_form_key_provider.phtml

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\Framework\View\Helper\SecureHtmlRenderer;
+
+$scriptString = <<<HTML
```

Comment:
his is not HTML, should be <<<JS probably.

Reply:
this should be `<<<script` and close with `script;`